src/custom/pages/App/index.tsx

```diff
@@ -16,6 +16,7 @@ import * as Sentry from '@sentry/react'
 import { Integrations } from '@sentry/tracing'
 import { version } from '@src/../package.json'
 import { environmentName } from 'utils/environments'
+import { useFilterEmptyQueryParams } from 'custom/hooks/useFilterEmptyQueryParams'
```

Comment:
Same thing here with `@src`. Unless you need to be explicit, don't prepend with `custom/`

```suggestion
import { useFilterEmptyQueryParams } from 'hooks/useFilterEmptyQueryParams'
```

But I know the IDE does that automatically